### PR TITLE
69 reference app shell

### PR DESCRIPTION
**Pull request by jsanford8**
This is the very basic holder for the ionic reference app. It does not pull in the plugins yet, or do anything with them. That will be added next.

This is just added in so we can get the automated build up and running.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/InternationalMixMob/CSCI-E-71_IMM_Slacker_Ref_App/pull/1" target="_blank">ReviewNinja

File: SlackerReferenceApp/bower.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "HelloIonic",
+  "private": "true",
+  "devDependencies": {
+    "ionic": "driftyco/ionic-bower#1.1.1"
```
Comment (frederickjansen) [inline SlackerReferenceApp/bower.json:5]: If ionic is a bower dependency, it should not be checked in

Comment (jsanford8) [inline SlackerReferenceApp/bower.json:5]: Bower was installed by Ionic, so I'm not sure if I get this dependency here. I can play around with removing it, if you think that is best

Comment (frederickjansen) [inline SlackerReferenceApp/bower.json:5]: I'd keep the dependency, just add www/libs to .gitignore. That way users can install using npm install && bower install.

Comment (jsanford8) [inline SlackerReferenceApp/bower.json:5]: on it, I'll update right now

Comment (jsanford8) [inline SlackerReferenceApp/bower.json:5]: updated now

Comment (frederickjansen) [inline SlackerReferenceApp/bower.json:5]: !fixed

**Pull request merged**